web/cobrands/angus/layout.scss

```diff
@@ -322,6 +322,10 @@ footer p.social a:active,footer p.social a:focus,footer p.social a:hover
   }
 }
 
+#main-nav {
```

Comment:
Yep, forgot to go back to the Angus cobrand once I'd added the omit_wrapper option :)